packages/back-end/src/services/experimentNotifications.ts

```diff
@@ -95,11 +103,18 @@ export const notifyAutoUpdate = ({
     type: "auto-update",
     triggered: !success,
     dispatch: () =>
-      dispatchEvent(context, experiment, {
-        type: "auto-update",
-        success,
-        experimentId: experiment.id,
-        experimentName: experiment.name,
+      dispatchEvent({
+        context,
+        experiment,
+        event: "warning",
+        data: {
+          object: {
+            type: "auto-update",
```

Comment:
auto-update isn't really a "warning" event type. Is this necessary to classify it as such?